test/indexer.py

```diff
@@ -1,11 +1,15 @@
 """For convenience, load test data into Elasticsearch index "test".
 
+Input must be JSON, not CSV. Unlike JSON, CSV values don't have types, so
```

Comment:
Beyond the scope of this test set, how do we expect most indexable to come in? As JSON exported from bigquery?

Reply:
No, the Elasticsearch index will have been created before `docker-compose up`. Added a comment to the top of this file ("For real datasets,..."). I also updated the [README.md](https://github.com/DataBiosphere/data-explorer/blob/master/README.md); I was missing a couple steps. Do things make sense now?